lib/require-patch.js

```diff
@@ -1,22 +1,13 @@
 'use strict'
 
 const Module = require('module')
-const glob = require('glob')
 const path = require('path')
 const ao = require('./index')
 
 const realRequire = module.constructor.prototype.require
 const patched = new WeakMap()
 const probes = {}
 
-// allow scoped packages to use same tests as unscoped versions. this
-// facilitates testing without duplicating for supported packages that
-// change to scoped.
-const nameToPatchFile = {
-  hapi: '@hapi/hapi',
-  vision: '@hapi/vision'
-}
-
```

Comment:
Does this impact whether the probe tests run for hapi?  Quick search I'm not seeing any test cases for either `hapi` or `@hapi` from the GH "Full Test" workflows... could be looking in the wrong place but want to double check.

Reply:
No. But...
The tests `hapi` and `vision` tests were indeed not running as they were not picked by the [test runner](https://github.com/appoptics/appoptics-apm-node/blob/d91896364bcc2a95e55321b471c7c594e3d74324/test.sh). That was "lost" in #197 (or followup rebases).  Fixed with 489392c6946961fbaddc3655d77cb32b15e6ce0b. See [Full Test Run](https://github.com/appoptics/appoptics-apm-node/runs/6284568075?check_suite_focus=true).

Reply:
👍